extensions/KHR/EGL_KHR_platform_wayland.txt

```diff
@@ -85,6 +85,11 @@ New Behavior
     belongs to Wayland. Any such call fails and generates an
     EGL_BAD_PARAMETER error.
 
+    Rendering to obtained EGLSurface or querying it with eglQuerySurface will
```

Comment:
Hmmmm, I think we should explicitly call out `EGL_BUFFER_AGE_KHR` as that's the only query I can think of. Querying, e.g., `EGL_WIDTH` and `EGL_HEIGHT` doesn't lock in a back buffer.

Reply:
I've updated wording.